Feature readme for hackathon submission (#6)
danielssonn/ethTO: README.md

![Cross Chain UX Rocks](assets/cross-chain-ux-rocks.svg)
![Powered by Axelar](assets/powered-by-axelar.svg)
![Made with SwingXYZ](assets/made-with-swing.xyz.svg)

![Runs On Moonbeam](assets/runs-on-moonbeam.svg)
![Runs On Ethereum](assets/runs-on-ethereum.svg)
![Runs On Polygon](assets/runs-on-polygon.svg)

> # The future is multi chain, but where is your wallet?

Reviewer: ```suggestion
# The future is multi-chain, but where is your wallet?
```

> ## Creating seamless cross chain NFT Renting experience

Reviewer: ```suggestion
## Creating a seamless cross chain NFT renting experience
```

<br><br>

<p>

  <a href="https://amplication.com" target="_blank">
    <img alt="amplication-logo" height="70" alt="Amplication Logo" src="https://amplication.com/images/amplication-logo-purple.svg"/>
  </a>
</p>

![Dashboard](assets/dashboard-main.png)

:point_right: [Our Glorious Deployment URL](https://here.xyz) :point_left:

---

Contents
===

 * [Why we built it](#why-we-built-it)
 * [What we built](#What-we-built)
 * [How we built it](#How-we-built)

 ### Why we built it

+ There are many blockchains out there, each with great assets and communities

> + The future is definitely multi chain!   

Reviewer: ```suggestion
+ The future is definitely multi-chain!   
```

> + However, the User eXperience interacting with multiple chains is cumbersome

Reviewer: ```suggestion
+ However, the User eXperience of interacting with multiple chains is cumbersome
```

>     + Mutiple wallets need to be created and navigated, by the user

Reviewer: ```suggestion
    + Multiple wallets need to be created and navigated by the user
```

    + Tokens need to be swapped around, also by the user

>     + It is so easy to make a mistake, loose on the opportunity, or simply get discouraged

Reviewer: ```suggestion
    + It is so easy to make a mistake, lose out on an opportunity, or simply get discouraged
```

 ### What we built

>  + Cross chain NFT renting experience

Reviewer: ```suggestion
 + Cross-chain NFT renting experience
```

 + With simple and straightforward UX
 + No matter what chain and which wallet or tokens you have 

>  + You can list and rent NFTs on multiple chains, without the multi chain hassle

Reviewer: ```suggestion
 + You can list and rent NFTs on multiple chains without the multi-chain hassle
```

 
 ### How we built it
 + We have created NFT renting contracts for lenders and renters
 + We have deployed the contracts on Polygon and Moonbeam
    - [Mumbai Polygonscan](https://mumbai.polygonscan.com/address/0x8c787c95e9f1bbc6153336571b7ab58cd57ad98c) 
    - [Moonbase Alpha](https://moonbase.moonscan.io/token/0xec19ebb094269b2782fbab3b5ce6e3cb4dea86a2)
    

>  + Mixed in some Axelar magic to make the contracts **cross chain BFF!**

Reviewer: ```suggestion
 + Mixed in some Axelar magic to make the contracts **cross-chain BFF!**
```
